scripts/monorepo/findRepoDeps.js

```diff
@@ -0,0 +1,63 @@
+const { spawnSync } = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const findGitRoot = require('./findGitRoot');
+
+function getAllPackageInfo() {
+  const gitRoot = findGitRoot();
+  const results = spawnSync('git', ['ls-tree', '-r', '--name-only', '--full-tree', 'HEAD']);
+  const packageInfo = {};
+
+  results.stdout
+    .toString()
+    .split('\n')
+    .map(line => line.trim())
+    .filter(line => line.endsWith('package.json'))
+    .forEach(packageJsonFile => {
+      const packageJson = JSON.parse(fs.readFileSync(path.join(gitRoot, packageJsonFile), 'utf-8'));
+      packageInfo[packageJson.name] = {
+        packagePath: path.dirname(packageJsonFile),
+        packageJson
+      };
+    });
+
+  return packageInfo;
+}
+
+function getDeps(packageJson) {
+  if (!packageJson) {
+    return [];
+  }
+
+  return [...Object.keys(packageJson.dependencies || {}), ...Object.keys(packageJson.devDependencies || {})];
```

Comment:
Might be better to do this in case there are dupes:
```suggestion
  return Object.keys({ ...(packageJson.dependencies || {}), ...(packageJson.devDependencies || {}) });
```